packages/tools/examples/livewireContour/index.ts

```diff
@@ -0,0 +1,180 @@
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  getRenderingEngine,
+} from '@cornerstonejs/core';
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  addDropdownToToolbar,
+  addButtonToToolbar,
+} from '../../../../utils/demo/helpers';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const {
+  LivewireContourTool,
+  PanTool,
+  ZoomTool,
+  StackScrollMouseWheelTool,
+  ToolGroupManager,
+  Enums: csToolsEnums,
+} = cornerstoneTools;
+
+const { ViewportType, Events } = Enums;
+const { MouseBindings } = csToolsEnums;
+const renderingEngineId = 'myRenderingEngine';
+const viewportId = 'CT_STACK';
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Livewire Tool',
+  'Interactive segmentation with intelligent scissors that uses Laplacian of Gaussian filter to find the shortest-path'
+);
+
+const content = document.getElementById('content');
+const element = document.createElement('div');
+
+// Disable right click context menu so we can have right click tools
+element.oncontextmenu = (e) => e.preventDefault();
+
+element.id = 'cornerstone-element';
+element.style.width = '500px';
+element.style.height = '500px';
+
+content.appendChild(element);
+
+const info = document.createElement('div');
+content.appendChild(info);
+
+const addInstruction = (text) => {
+  const instructions = document.createElement('p');
+  instructions.innerText = `- ${text}`;
+  info.appendChild(instructions);
+};
```

Comment:
Can you make this a general utils that you import here? thanks

Reply:
Done